ts/auto-import.ts

```diff
@@ -52,40 +54,40 @@ export default class AutoImport{
         // decides which ones to include in which bundles
         let splitter = new Splitter({
             analyzers: this.analyzers,
-            bundles: ['app', 'tests'],
-            bundleForPath(path) {
-                if (testsPattern.test(path)) {
-                    return 'tests';
-                } else {
-                    return 'app';
-                }
-            }
+            bundles,
+            bundleForPath
         });
 
         // The Bundlers ask the splitter for deps they should include and
         // are responsible for packaging those deps up.
+        return bundles.map(bundle =>
+          new Bundler({
+            outputFile: `ember-auto-import/${bundle}.js`,
+            splitter,
+            bundle,
+            environment: this.env,
+            packages: this.packages,
+            consoleWrite: this.consoleWrite
+          })
+        );
+    }
 
-        let appBundler = new Bundler({
-          outputFile: `ember-auto-import/app.js`,
-          splitter,
-          bundle: 'app',
-          environment: this.env,
-          packages: this.packages,
-          consoleWrite: this.consoleWrite
-        });
+    private makeTree() {
+      if (!this.tree) {
+        this.tree = debugTree(new MergeTrees(this.bundlers.map(b => b.tree)), 'output');
+      }
+      return this.tree;
+    }
 
-        let testsBundler = new Bundler({
-          outputFile: `ember-auto-import/test.js`,
-          splitter,
-          bundle: 'tests',
-          environment: this.env,
-          packages: this.packages,
-          consoleWrite: this.consoleWrite
-        });
-        return [appBundler, testsBundler];
+    treeForVendor(){
+      return this.makeTree();
     }
 
-    treeForVendor(tree){
-        return new MergeTrees([tree].concat(this.bundlers.map(b => b.tree)).filter(Boolean));
+    treeForPublic() {
+      return debugTree(new Funnel(this.makeTree(), {
+        srcDir: 'ember-auto-import',
```

Comment:
I wonder if the folder name could be something more generic instead of `ember-auto-import`, as this will be part of the url for these files.  Maybe something like `chunks`?

Reply:
`srcDir` is where `Funnel` reads from, not where it writes to. At the moment I'm storing the chunks directly in the `assets` folder, named like `chunk.0.js`, `chunk.1.js`.